src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const newObj = {
+    result: 0,
+
+    reset() {
+      newObj.result = 0;
+
+      return this;
+    },
+
+    operate(callback, value) {
+      callback(value);
```

Comment:
In the 'operate' method, you should pass 'this' as a context to callback function. Correct way to call it would be 'callback.call(this, value)'.